src/undate/dateformat/base.py

```diff
@@ -15,9 +15,8 @@
 import importlib
 import logging
 import pkgutil
-from typing import Dict, Type
 from functools import lru_cache  # functools.cache not available until 3.9
```

Comment:
We could use this now, right? We dropped 3.8 support?

Reply:
thanks for catching - although not sure why this was in the change set, sorry for such an enormous PR!